prisma/schema.prisma

```diff
@@ -15,3 +15,45 @@ model sessions {
   expires DateTime
   session Json?
 }
+
+model users {
+  id String @id @default(auto()) @map("_id") @db.ObjectId
+  uid String @unique
+  access String[] @default([])
+  bad_pwd_count Int?
+  blocked Boolean?
```

Comment:
should this default to false?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/237/commits/991410c2b7d76f49852b5c3ebe79dfa0bcb18e36